### PR TITLE
update script for runninglinuxkernel_4.0

### PR DESCRIPTION
**Pull request by miraclemore**
1. Add script for qemu boot kernel command
2. Enable virt-9p to support folder shared
3. add /mnt and /lib/modules/4.0.0+ for rootfs
4. Currently only support x86 platform
5. fix some build warnning

### CODE REVIEW

File: _install_x86/etc/init.d/rcS [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
 mkdir -p /tmp
 mkdir -p /proc
+mkdir -p /mnt
+mkdir -p /lib/modules/4.0.0+
 /bin/mount -a
 mkdir -p /dev/pts
 mount -t devpts devpts /dev/pts
 echo /sbin/mdev > /proc/sys/kernel/hotplug
 mdev -s
+mount -t 9p -o trans=virtio kmod_mount /mnt
```
Comment (figozhang) [inline _install_x86/etc/init.d/rcS]: it had better do this mount by yourself , maybe some guys have not insmod those 9p kernel modules

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    DBG="-s -S"
+fi
+
+case $1 in
+    x86_64)
+        qemu-system-x86_64 -kernel arch/x86/boot/bzImage \
+                           -append "rdinit=/linuxrc console=ttyS0" -nographic \
+                           --virtfs local,id=kmod_dev,path=$PWD/kmodules,security_model=none,mount_tag=kmod_mount \
+                           -enable-kvm \
+                           $DBG ;;
```
Comment (figozhang) [inline run.sh]: some error when i run your patch:

figo@figo-OptiPlex-9020:~/work/test1/runninglinuxkernel/runninglinuxkernel_4.0$ sh run.sh x86_64
qemu-system-x86_64: --virtfs local,id=kmod_dev,path=/home/figo/work/test1/runninglinuxkernel/runninglinuxkernel_4.0/kmodules,security_model=none,mount_tag=kmod_mount: 9pfs Failed to initialize fs-driver with id:kmod_dev and export path:/home/figo/work/test1/runninglinuxkernel/runninglinuxkernel_4.0/kmodules

File: arch/x86/configs/x86_64_defconfig [hunk 3]
```diff
@@ -3681,3 +3681,10 @@ CONFIG_FONT_SUPPORT=y
 CONFIG_FONT_8x8=y
 CONFIG_FONT_8x16=y
 CONFIG_ARCH_HAS_SG_CHAIN=y
+
+CONFIG_NET_9P=y
+CONFIG_NET_9P_VIRTIO=y
+CONFIG_9P_FS=y
+CONFIG_9P_FS_POSIX_ACL=y
+CONFIG_PCI=y
+CONFIG_VIRTIO_PCI=y
```
Comment (figozhang) [inline arch/x86/configs/x86_64_defconfig]: i like those CONFIG build with kernel modules.

other warning:
arch/x86/configs/x86_64_defconfig:3685:warning: override: reassigning to symbol NET_9P
arch/x86/configs/x86_64_defconfig:3689:warning: override: reassigning to symbol PCI
arch/x86/configs/x86_64_defconfig:3690:warning: override: reassigning to symbol VIRTIO_PCI

File: _install_x86/etc/init.d/rcS (hunk 1 above)
Comment (miraclemore) [reply _install_x86/etc/init.d/rcS]: Alright! I consider that new comers don't know how to mount that. So I add the command to rcS. Even someone doesn't insmod the module for 9p, it doesn't influence the minimal rootfs to boot up. And also the defconfig is configured that to y as default value.

File: run.sh (hunk 2 above)
Comment (miraclemore) [reply run.sh]: Due to it is my first github commit. So some mistake caused by move the commit to fork's one.
just ignore that. I will create a new pull request for that.

**Comment by miraclemore** [comment]
Close due to some mistakes!
I will fix that and then create a new one.

**Pull request closed without merging**